### PR TITLE
ICDS: New Child Monthly Data Source

### PR DESCRIPTION
**Pull request by sheelio**
@emord 
This should get added to the existing monthly queue.  (Probably best reviewed commit by commit).

### CODE REVIEW

File: custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json [hunk 1]
```diff
@@ -2415,6 +2622,201 @@
...
+        "type": "switch",
+        "switch_on": {
+          "type": "nested",
+          "argument_expression": {
+            "aggregation_fn": "last_item",
+            "type": "reduce_items",
+            "items_expression": {
+              "type": "icds_get_case_forms_by_date",
+              "xmlns": [
+                "http://openrosa.org/formdesigner/7a55754119359466b1951d7251068bd4f45e73c3"
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json]: How many times does this form get submitted per case? If it's a lot, this is probably going to be really expensive

File: custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json [hunk 2]
```diff
@@ -102,25 +102,24 @@
...
         "display_name": null,
         "expression": {
-          "type": "nested",
-          "argument_expression": {
-            "type": "ancestor_location",
-            "location_id": {
-              "expression": {
-                "property_name": "owner_id",
-                "type": "property_name"
+          "location_id_expression": {
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json]: Why did you need to revert ancestor location?

**Review by emord — COMMENTED** [review]
Need to do figure out how many shards to give this index. How many docs do you anticipate this to create at scale?

**Comment by sheelio** [comment]
@emord 
This is the same as the other child health monthly (which we will eventually remove).  Approximately 300 per AWC, so 51 million for 170K AWCs.

I reverted the ancestor location changes since they weren't working when I tested them locally (and the previous configuration using location_parent_id was working).

File: custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json (hunk 1 above)
Comment (sheelio) [inline custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json]: About once a month.  Over a child's life this form should be filled out approximately 30 times.

File: custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json [hunk 3]
```diff
@@ -2373,6 +2371,35 @@
           }
         ]
       },
+      "height_eligible": {
+        "type": "and",
+        "filters": [
+          {
+            "type": "boolean_expression",
+            "operator": "gte",
```
Comment (sheelio) [inline custom/icds_reports/ucr/data_sources/child_health_cases_monthly_tableau2.json]: There isn't a between filter unfortunately (at least not that I know of).
